core/staking/checkpoint_test.go

```diff
@@ -79,7 +79,7 @@ func TestCheckpointLoadNoDuplicates(t *testing.T) {
 
 	cp := &checkpoint.Checkpoint{}
 	if err := proto.Unmarshal(cpFile, cp); err != nil {
-		println(err)
+		t.Fatal(err)
```

Comment:
println!=Fatal ?

Reply:
Sure, but that's a test. `println` was most likely a left over. In this context it's very likely that failing the test is the expected behavior.